### ISSUE TITLE
Performance of CoraNet with different ratios of labeled to unlabeled samples

### ISSUE THREAD

**Issue by nguyenpbui** [koncle/CoraNet#8]
Dear authors,
I have been studying your article. I have a concern with Table III. The values of DSC (%) with different ratios of labeled to unlabeled samples.
You use three ratios, i.e., 1:4, 1:2, 1:1. For these three ratios, did you use the same number of labeled samples or you divide the whole dataset with these ratios?
For example, we have a dataset with 1000 images, with three ratios above and 100 labeled samples, the corresponding samples are: 100:400, 100:200, and 100:100.
If I was wrong, please correct me.
Thank you

**Comment by koncle** [comment]
Thank you for your interest. We divide the whole training set to these ratios and leave the test set unchanged.  Therefore the performance of 1:1 is higher than the others.

**Comment by nguyenpbui** [comment]
I got it.
Thank you.
